### DISCUSSION TITLE
Notifications cause audio issue for phone calls

### DISCUSSION THREAD

**Discussion by ruckusvol** [BimmerGestalt/AAIdrive#607] [General]
I'm on the lastest version without map.  Today I was on phone calls and twice when a notification would come in the bluetooth phone audio would turn into a robot voice that was impossible to understand.  The first time I thought it was just the call quality.  But later, after the car had been restarted and on a different call another notification caused the same issue.  Anyone else experience this or have an idea what could cause it?  

I just moved to the no map version a couple days ago.  I haven't had this problem before.  I'm on a Pixel 6 on Android 13.

**Comment by hufman** [comment]
That's very unusual! AAIdrive itself doesn't manage anything about the Bluetooth audio connection, it just controls music apps over the OS-managed Bluetooth connection. The only audio-related option is the Notification sound, which simply plays a sound file when a new notification is detected, and this code hasn't changed in a long time. Does this issue still occur if this notification sound option is disabled?
Does the problem happen if AAIdrive isn't installed? Android 13 just came out 2 weeks ago, perhaps this upgrade coincided with your change to a different AAIdrive version.

**Reply by ruckusvol** [reply]
Thanks.  I figured it probably wasn't related to AAIDRIVE but figured this was the best place to ask.  The app is awesome.  I disabled the sound for the AAIDRIVE notification in Android and we'll see how it does.  If not it probably is A13 instead of the app.
